### PR TITLE
Fix docker-compose - Use variables to set docker postgres tag

### PR DESCRIPTION
**Pull request by dgiorgio**

### CODE REVIEW

File: installer/roles/local_docker/defaults/main.yml [hunk 1]
```diff
@@ -10,12 +10,10 @@ rabbitmq_port: "5672"
 rabbitmq_user: "guest"
 rabbitmq_password: "guest"
 
-postgresql_version: "10"
+postgresql_version: "10-alpine"
```
Comment (shanemcd) [inline installer/roles/local_docker/defaults/main.yml]: Hi and sorry for the slow review. Does this 'just work' if you had the old image running and re-ran a newer installer? I'd like to test this first, but I don't have time to simulate an upgrade right now, as I'm leaving for vacation after tomorrow. I would merge this today if you reverted to the default value and specify the image you want on your side when running the installer.

Comment (dgiorgio) [reply installer/roles/local_docker/defaults/main.yml]: done

**Comment by ryanpetrello** [comment]
recheck

**Review by shanemcd — APPROVED** [review]
Sorry for the delay on this one too. Merging now.

**Pull request merged**